packages/core/src/composeAPI.ts

```diff
@@ -9,7 +9,7 @@ import { createSignRsaSignature } from './createSignRsaSignature';
 import { createVerifyRsaSignature } from './createVerifyRsaSignature';
 
 export type Settings = {
-  providers?: IriProviders;
+  provider?: IriProvider;
```

Comment:
I don't think `IriProvider` is relevant name. It can be `TangleProvider` and/or `IotaProvider`. I prefer the later. Let's create a new pull request.